src/test/java/org/jboss/xavier/integrations/route/MainRouteBuilder_DirectAddUsernameHeaderTest.java

```diff
@@ -43,7 +43,7 @@ public void mainRouteBuilder_routeDirectAddUsernameHeader_NoContentGiven_ShouldA
         });
 
         //Then
-        assertThat(result.getIn().getHeader("analysisUsername")).isEqualTo("");
+        assertThat(result.getIn().getHeader("analysisUsername")).isEqualTo(null);
```

Comment:
Please move it to be (also in the cases below):
```suggestion
        assertThat(result.getIn().getHeader("analysisUsername")).isNull();
```